sceptre/cli/describe.py

```diff
@@ -0,0 +1,151 @@
+import click
```

Comment:
Name of this file is slightly misleading, since it contains all of the "read only" operations, instead of just the describe commands.

Reply:
Yeah - so realised I can't split out "list" commands in to a list.py and import list, because it clashes with built-in python keyword "list". Not sure what to rename too?

Reply:
Creating extra list.py file

Reply:
Expanding on this, we discussed offline that we can just do

```python
from list import list_group
```
for the purposes of using that module within Sceptre. It is more verbose, but it makes the structure of the commands more intuitive and, thus, more maintainable.

This may cause confusion if someone wanted to use Sceptre as a Python module and import the Sceptre list.py; however, we don't officially support that and users may sidestep this issue by adding the import statement:

```python
import list as list_commands
```

instead.